exasol_error_reporting_python/parameters_mapper.py

```diff
@@ -0,0 +1,35 @@
+from exasol_error_reporting_python.placeholder_handler import PlaceholderHandler
+
+
+class ParametersMapper:
+    def __init__(self, text: str, arguments: str):
+        self._text = text
+        self._parameters = arguments
+        self._parameter_idx = 0
+        self._parameter_dict = {}
+
+    @classmethod
+    def get_params_dict(cls, text, arguments):
+        parameters_mapper = cls(text, arguments)
+        parameters_mapper._map_parameters()
+        return parameters_mapper._parameter_dict
+
+    def _map_parameters(self):
```

Comment:
what exactly does this function. My guess is, that for all placeholders are parameters available, but what is if the parameters are not defined in the direct argument list. For example, https://github.com/exasol/error-reporting-python/blob/7f581760e5a01917569edfa8aa028d50722a11d2/exasol_error_reporting_python/error_message_builder.py#L43  you only use the argument given by the _add_parameters call and not the already selected one. However, it could also be the case, that you use the placeholders to filter which parameters to add to parameter_dict. Please explain the function.

Reply:
parameters for the placeholder can be given two way:

1. call parameter function. eg: `message("{{p1}} and {{p2}}").parameter("p1", "parameter_1").parameter("p2", "parameter_2")`

2. give parameters inline. eg: `message("{{p1}} and {{p2}}", "parameter_1", "parameter_2")`

this method is for mapping inline parameters that is second way. 

Is that clear now?

Reply:
ah ok the second way are positional arguments, I think, I slowly understand it, thx

Reply:
Please add, a docstring to the function and explain that it looks up positional arguments